lib/clientsidescripts.js

```diff
@@ -36,6 +43,83 @@ function wrapWithHelpers(fun) {
       '  return (' + fun.toString() + ').apply(this, arguments);');
 }
 
+/* Tests if an ngRepeat matches a repeater
+ *
+ * @param {string} ngRepeat The ngRepeat to test
+ * @param {string} repeater The repeater to test against
+ * @param {boolean} exact If the ngRepeat needs to be more or less exactly the
```

Comment:
What about `If the ngRepeat expression should exactly match`.... it's not 100% precise because of `track by` but I think it gets the idea across more clearly than "more or less"